### DISCUSSION TITLE
Allow disabling entry creation from relationship fields

### DISCUSSION THREAD

**Discussion by FreekVR** [craftcms/cms#9030] [Ideas]
### Description

In some cases, it might not be desirable to allow users to create new entries and only allow them to select an existing one from the entry field modal -- even if they do have permissions to do so in other places. This would simplify the workflow a bit for inexperienced users.

As such, would it be possible to add an optional "Allow creating new entries in modal" checkbox to the relationship field settings?

**Comment by nickdunn** [comment]
Oh yes please! I'm struggling right now with a client who is creating new entries in this way, and the little modal form that pops up from the picker modal really doesn't scale well for complex section/field arrangements, and they're all sorts of confused. I'd much rather be able to hide the "create new" form completely and train them to go elsewhere to create the entries.
